### PR TITLE
chore: additional quotes, fix typographical errors, add nsfw tag

### PR DESCRIPTION
**Pull request by srosebattles**
Clearly, I lied when I said my most recent PR was going to be my last one for a while. In this PR:
-Add brand-new quotes (one more by Chris Walley and several from Ray Bradbury), as well as utilize some existing quotes for other times. Removed I think one from a.m. because context clearly indicated it was a p.m. quote.
-It bothered me that we had near-identical quotes from The Curious Incident of the Dog in the Night-Time for both seven minutes before and after midnight; verified that the seven minutes after midnight is the correct version.
-Add nsfw tag to a few more suggestive quotes and ones with milder curses ("damn it" and "bastard") that didn't trigger my attention last time.
-Trim a few quotes that were longer than they needed to be.
-Fix some additional typographical errors.

Hopefully I didn't inadvertently break anything this time!

### CODE REVIEW

File: litclock_annotated.csv [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
 00:00|midnight|It was an animal thing. A wilderness thing. Nothing about her belonged to her previous self in that moment. Above all her voice. This could have been the high shriek of a hawk, the soul-haunting howl of a wolf, the rasping cry of a red fox at midnight. It could have been any of them, but not the scream of a sixteen-year-old schoolgirl.|The Island of Missing Trees|Elif Shafak|unknown
 00:00|midnight|Oliver Dubois rammed a pickup truck through his front door on a sweaty midnight in July, at the tail end of a massive block party. <br>It wasn’t his truck, and the street was closed to traffic, so the vehicle’s emergence out of darkness and slow rumble up Oak Drive scattered neighbors and guests alike into confused groups on candlelit lawns.|Behind the Lie|Emilya Naymark|unknown
-00:00|midnight|It was near midnight when she left the Palace in a clothes-hamper borne by two porters, covered with her own soiled smocks and petticoats which were supposedly being carried to the laundress.|Forever Amber|Kathleen Windsor|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:13]: Moved this, since it says "near midnight"

File: litclock_annotated.csv [hunk 2]
```diff
@@ -33,24 +32,23 @@
...
 00:00|midnight|We have heard the chimes at midnight.|Henry IV|William Shakespeare|unknown
 00:00|midnight|It was surely midnight by now. A lone bird called, and the individual trees melted together in one forbidding and unfriendly mass. To Savonius they seemed to incarnate this dismal adventure which had been thrust on him against his will. Somewhere in this deep forest lay this demented man he was supposed to try to comfort and calm.|The Hammer of God|Bo Giertz|unknown
 00:00|midnight|There was the midnight hike with the boys to the end of the promenade and the big argument about the election of a housemaster. What had become of responsibility and service? And brotherly love?|The Hammer of God|Bo Giertz|unknown
-00:00|midnight|It was near midnight. <br/>Torvik took the doll which had been left on the couch in the study, after first having mended its broken foot.|The Hammer of God|Bo Giertz|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:53]: Moved this, since it says "near midnight"

File: litclock_annotated.csv [hunk 3]
```diff
@@ -937,18 +945,18 @@
...
+09:42|9:42|9:36<br/>Fuck. I cannot make this bed. Why won't this sheet lie flat?<br/>9:42<br/>And why do they make mattresses so heavy?|The Undomestic Goddess|Sophie Kinsella|nsfw
 09:45|a quarter to ten|At a quarter to ten on the morning of August 15th, Leandro crossed into the town of Troy. His stomach was tight with anticipation and — let’s face it, folks — a tingle of fear. His skin felt cold.|The Tommyknockers|Stephen King|unknown
 09:45|quarter to ten|She looked at the clock on the night table and saw it was quarter to ten. She’d slept another two hours. For a moment she was alarmed; maybe she’d suffered a concussion or a fracture after all.|Full Dark, No Stars|Stephen King|unknown
-09:45|twenty-five minutes to ten|“At twenty-five minutes to ten he was tapping on the closed door of Blore’s room.<br>The latter opened it cautiously. His hair was tousled and his eyes were still dim with sleep.”|And Then There Were None|Agatha Christie|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:951]: Moved this, it was at the wrong time

File: litclock_annotated.csv [hunk 4]
```diff
@@ -33,24 +32,21 @@
...
 00:00|twelve|I took her hand in mine, and bid her be composed; for a succession of shudders convulsed her frame, and she would keep straining her gaze towards the glass. 'There's nobody here!' I insisted. 'It was YOURSELF, Mrs. Linton: you knew it a while since.' 'Myself!' she gasped, 'and the clock is striking twelve! It's true, then! that's dreadful!'|Wuthering Heights|Emily Brontë|unknown
 00:00|midnight|I was born in the city of Bombay . . . On the stroke of midnight, as a matter of fact. Clock-hands joined palms in respectful greeting as I came. Oh, spell it out, spell it out: at the precise instant of India's arrival at independence, I tumbled forth into the world.|Midnight's Children|Salman Rushdie|unknown
 00:00|midnight|It is midnight. The rain is beating on the windows. I am calm. All is sleeping. Nevertheless I get up and go to my desk. I can't sleep.|Molloy|Samuel Beckett|unknown
-00:00|midnight|It was nearing midnight and the Prime Minister was sitting alone in his office, reading a long memo that was slipping through his brain without leaving the slightest trace of meaning behind.|Harry Potter and the Half-Blood Prince|JK Rowling|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:43]: Moved

File: litclock_annotated.csv [hunk 5]
```diff
@@ -33,24 +32,21 @@
...
 00:00|midnight|Midnight had come upon the crowded city. The palace, the night-cellar, the jail, the madhouse; the chambers of birth and death, of health and sickness; the rigid face of the corpse and the calm sleep of the child - midnight was upon them all.|Oliver Twist|Charles Dickens|unknown
-00:00|midnight|Midnight is approaching, and while the peak of activity has passed, the basal metabolism that maintains life continues undiminished, producing the basso continuo of the city's moan, a monotonous sound that neither rises or falls but is pregnant with foreboding.|After Dark|Murakami|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:45]: Moved

File: litclock_annotated.csv [hunk 6]
```diff
@@ -1790,53 +1806,55 @@
...
 18:00|six o’clock|She sounded surprised, and possibly also disappointed. He said, “Listen, I can’t speak for long, so please concentrate on what I’m about to say. I want you to pack a week’s worth of clothes and get the garage to drive you and the children to your parents’ right away.”<br>“But it’s six o’clock.”<br>“They’ll still be open.”<br>“Why do we have to go in such a rush? What’s happened?”<br>“Nothing. Nothing yet, anyway. I just want to know you’re somewhere safe.”<br>“It sounds rather panicky. I hate people who panic.”|Munich|Robert Harris|unknown
-18:00|six|The next morning, Sister Veronica is still at prayer in the chapel. At six, when the other sisters file in for Vigils, she is there, prostrate before the cross, her arms outstretched, her forehead touching the cold stone tile. It is only when they lean forward to touch her arm gently that the women see that the blood has settled in her face. She has been dead for many hours.|The Power|Naomi Alderman|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1819]: moved

File: litclock_annotated.csv [hunk 7]
```diff
@@ -2189,33 +2212,33 @@
 22:00|ten o’clock|Meanwhile, the blood of the city’s saints had taken on a whole range of properties hitherto unknown to science. One lady’s was liquefying at precisely ten o’clock every Tuesday, while San Giovanni’s bubbled away obligingly whenever it heard Holy Scripture.|The Wedding Officer|Anthony Capella|unknown
 22:00|2200 hours|He glanced at the clock. 2200 hours. Getting on toward midnight. He hurried to dress.|The Andromeda Strain|Michael Crichton|unknown
 22:00|ten|By ten, Quoyle was drunk. The crowd was enormous, crushed together so densely that Nutbeem could not force his way down the hall or to the door and urinated on the remaining potato chips in the blue barrel, setting a popular example.|The Shipping News|E. Annie Proulx|unknown
-22:00|ten o'clock|Her body asserted itself with a restless movement of the knee, and she stood up. 'Ten o'clock,' she remarked, apparently finding the time on the ceiling. 'Time for this good girl to go to bed.'|The Great Gatsby |F. Scott Fitzgerald|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:2192]: duplicate

**Comment by JohannesNE** [comment]
Again. Thanks for putting a lot of work into this. I'm sorry it took so long for me to find time to look at it.

**Pull request merged**